src/luarocks/static_flags.lua

```diff
@@ -0,0 +1,41 @@
+
+local static_flags = {}
+package.loaded["luarocks.static_flags"] = static_flags
+
+local util = require("luarocks.util")
+local index = require("luarocks.index")
+local fetch = require("luarocks.fetch")
+local deps = require("luarocks.deps")
+local search = require("luarocks.search")
+
+util.add_run_function(static_flags)
+
+--luarocks static-flags luasec
+function static_flags.command(flags, rockspec_file)
+   assert(type(rockspec_file) == "string" or not rockspec_file)
+   local rockspec = fetch.load_rockspec(rockspec_file)
+   local command = ""
+
+   -- Do I need to fulfill deps? like this?
+   -- local ok, err, errcode = deps.fulfill_dependencies(rockspec, "all")
+   -- if err then return nil, err, errcode end
+
+   if #rockspec.dependencies > 0 then
+      for _, dep in ipairs(rockspec.dependencies) do
+         if not dep.name:match("lua$") then
```

Comment:
This needs to be recursive. i.e. we also need flags of dependencies of dependencies of dependencies